config/owasp/suppressions.xml

```diff
@@ -8,11 +8,13 @@
       CVE-2022-1471 - SnakeYaml's Constructor() class does not restrict types which can be instantiated during deserialization, spring uses SafeConstructor which is the recommended workaround
       CVE-2022-40705 - see https://github.com/jeremylong/DependencyCheck/issues/5543
       CVE-2022-45688 - A stack overflow in the XML.toJSONObject component of hutool-json, hutool is not in the dependencies list when ./gradlew dependencies is run
+      CVE-2023-34411
```

Comment:
Are there any links/notes that can be added to say why it's a false positive?

Reply:
not yet the moment

Reply:
Is the following link the evidence that it's a false positive? https://github.com/jeremylong/DependencyCheck/issues/5771

If so, can that be included?